src/styles/validation-errors-card.scss

```diff
@@ -0,0 +1,186 @@
+$validation-errors-border-radius: 8px;
+
+.ob-validation-notification-wrapper {
+  position: fixed;
+  bottom: 1rem;
+  right: 1rem;
+  z-index: 31;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 300ms;
+}
+
+.ob-validation-color-transition {
+  transition-property: color;
+  transition-duration: inherit;
+  transition-timing-function: inherit;
+}
+
+.ob-validation-notification-card {
+  transition-property: transform, background-color;
+  transition-duration: inherit;
+  transition-timing-function: inherit;
+  border-radius: $validation-errors-border-radius;
+  max-height: 350px;
+  overflow-y: auto;
+  border-color: $danger;
+  border-width: 2px;
+  border-style: solid;
+  animation: pulse-animation 1.5s infinite;
+  @keyframes pulse-animation {
+    0% {
+      box-shadow: 0 0 0 0px rgba(255, 0, 0, 0.4);
+    }
+    100% {
+      box-shadow: 0 0 0 16px rgba(255, 0, 0, 0);
+    }
+  }
+}
+
+// Card
+.ob-validation-notification-card.is-expanded {
+  background-color: $white;
+  color: $black;
+}
+
+.ob-validation-notification-card.is-contracted {
+  background-color: $danger;
+  color: $white;
+}
+
+// Card Content
+.ob-validation-notification-card-content {
+  transition-property: padding;
+  transition-duration: inherit;
+  transition-timing-function: inherit;
+  padding: 1rem;
+  padding-bottom: 1.5rem;
+}
+.is-contracted .ob-validation-notification-card-content {
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+// Card Content Header
+.ob-validation-notification-card-header-wrapper {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.ob-validation-notification-card-header-title-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.ob-validation-notification-card-header-title-text {
+  font-weight: $weight-semibold;
+}
+
+.ob-validation-notification-card-header-title-icon {
+  margin-right: 0.5rem;
+  font-size: 22px;
```

Comment:
Does this (and some other hard coded values) have pre-existing variables we can use?

Reply:
Mm, not really, theres existing icon size classes, but small is 18 and medium is like 30, so I couldn't think of a class name to go between the two, so I just hardcoded it.

Reply:
Some of the rem stuff could probably be replaced with the `$sizeX` sass variables, though.

Reply:
Happy to skip over the font-size, up to you if you want to replace any of the hardcoded rems with variables 👍

Reply:
I'll add a task for it in jira.